src/components/body/Body.tsx

```diff
@@ -0,0 +1,49 @@
+//components
+import {Home} from "./components/home";
+import {Chat} from "./components/chat";
+import {Teams} from "./components/teams";
+import {Activity} from "./components/activity";
+import {Apps} from "./components/apps";
+import {Calls} from "./components/calls";
+import {Files} from "./components/files";
+import {Help} from "./components/help";
+import {More} from "./components/more";
+import {SprinklrHub} from "./components/sprinklrHub";
+
+//styles
+import "./styles.css";
+
+enum ViewType {
+  Home = 'Home',
+  Activity = 'Activity',
+  Chat = 'Chat',
+  Teams = 'Teams',
+  Calls = 'Calls',
+  Files = 'Files',
+  SprinklrHub = 'Sprinklr Hub',
+  More = 'More',
+  Apps = 'Apps',
+  Help = 'Help',
+}
```

Comment:
Move this to a constant file on same level.

Always keep constants all caps.

enum VIEW_TYPES {
    HOME = 'HOME
}